.github/workflows/ubuntu.yml

```
@@ -0,0 +1,90 @@
+name: Ubuntu
+
+on:
+  push:
+    branches:
+      - fork/*
+      - master
+  pull_request:
+    branches:
+      - fork/*
+      - master
+  release:
+    types: [created]
+
+env:
+  CTEST_OUTPUT_ON_FAILURE: 1
+
+jobs:
+  build:
+    if: "!contains(github.event.head_commit.message, 'skip ci') && !contains(github.event.head_commit.message, 'ci osx') && !contains(github.event.head_commit.message, 'ci win')"
+    
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        nginx: [ '1.21.0', '1.21.1' ]
```

Why don't we use latest stable release too? (and latest mainline release)
```suggestion
        nginx: [ '1.22.0', '1.21.6' ]
```

---

I just saw the PR date and understood why those were the versions :)